### ISSUE TITLE
There are some problems with HiararchicalATT.py

### ISSUE THREAD

**Issue by ghost** [Cheneng/HiararchicalAttentionGRU#1]
model/HiararchicalATT.py
# **************************#
# word level GRU
        x, _ = self.word_GRU(x)
        Hw = F.tanh(self.w_proj(x))
        w_score = self.softmax(Hw.matmul(self.w_context_vector))
        x = x.mul(w_score)
        x = torch.sum(x, dim=1)
1. Why do we sum up X at last? could we ''concat'' it?
2.The final result x [batch_size, hidden_size*2] (Am I right?), 
how to spread it to _align_sent(input_matrix, sent_num, sent_max=None) and input_matrix is [sentence_num, sentence_embedding], where batch_size ? why sentence_num?
# **************************#
